src/routes/settings.tsx

```diff
@@ -48,29 +60,46 @@ function Settings() {
           </CardContent>
         </Card>
 
+        {/* Books Per Search Input */}
         <Card>
           <CardHeader>
             <CardTitle>Books per Search</CardTitle>
             <CardDescription>Set the maximum number of books displayed per search.</CardDescription>
           </CardHeader>
           <CardContent>
-            <Input type="number" value={booksPerSearch} onBlur={(e) => setBooksPerSearch(Number(e.target.value))} min={1} max={100} />
+            <input
```

Comment:
Why are we not using the Input component?

Reply:
searchIcon

Reply:
Which is why you don't override the global Input component and put a search icon in there, see https://github.com/bookracy/frontend/pull/5#discussion_r1720943053